Makefile

```diff
@@ -30,7 +30,7 @@ simulator: ${UNIX_DIR}/build-simulator/micropython
 
 .PHONY: flash
 flash: build
-	esptool.py -p /dev/ttyACM* -b 460800 --before default_reset --after no_reset \
+	esptool.py -p /dev/cu.usbmodem1101 -b 460800 --before default_reset --after no_reset \
```

Comment:
Please revert this.

Reply:
I wonder what the best way to handle this is. I guess conditionals based on the operating system? 

I have to change it on my machine as well (also a macbook) to get this to flash. Probably not worth resolving in this review, but it is perhaps worth opening a separate issue to make the Makefile more robust to this.